versioned_docs/version-2.0/how_to_guides/playground/custom_tls_certificates.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+
+# Use custom TLS certificates
+
+:::note Self-hosted enterprise only (version 0.6 and later)
+
+This feature is only available for self-hosted enterprise customers, starting from version 0.6.
+
+If you are interested in this plan, please contact sales@langchain.dev for more information.
+
+:::
+
+You can use custom TLs certificates to connect to model providers in the LangSmith playground. This is useful if you are using a self-signed certificate or a certificate from a custom certificate authority.
```

Comment:
TLS not TLs

Reply:
also maybe mention need to use mutual TLS

Reply:
I think we also need to note that this will apply to all your Azure models(so its kind of breaking)